NEWS.md

```diff
@@ -1,5 +1,11 @@
-## 2019-03-21 v1.3.0-SNAPSHOT
+## 2019-05-02 v1.3.0-SNAPSHOT
 * Added Spring DI support
+* Added dependency on users interface
```

Comment:
Do we have dependency on mod-users in module-descriptor ('requires' section)?

Reply:
yep) Oleksii added the dependency, but did not update the News